public/components/agents/fim/inventory/registryValues/registryValues.tsx

```diff
@@ -46,23 +47,25 @@ export const RegistryValues = (props) => {
       field: 'date',
       name: 'Date',
       sortable: true,
+      render: emptyFieldHandler()
     },
     {
       field: 'value',
       name: 'Value name',
       sortable: true,
-      render: (item) => item.name,
+      render: emptyFieldHandler((item) => item.name),
```

Comment:
remove please `emptyFieldHandler` from all but this.
we'll only notify this field when is empty.